lib/MusicBrainz/Server/Edit/Relationship/Edit.pm

```diff
@@ -91,6 +91,8 @@ has '+data' => (
         relationship_id => Int,
         type0 => Str,
         type1 => Str,
+        entity0_credit => Optional[Str],
+        entity1_credit => Optional[Str],
```

Comment:
I’m not sure about using`Optional` over `Nullable` here. In both cases, tests need to be updated.

Reply:
I think `Optional` makes more sense since those aren't nullable in the DB. (`Nullable` is just defined as `Optional[Maybe[x]]`.)